_guides/software-request-guide.md

```diff
@@ -0,0 +1,89 @@
+
+### Research
+Once you have identified a need, discuss this with your supervisor and colleagues to get more information about what tool may be best to use. Be sure check if the software you want is available in the [GSA EA Analytics & Reporting](https://ea.gsa.gov/#!/itstandards) (GEAR) IT Standards list. Note: The GEAR site is only available within the GSA network. Once you have identified the software you would like, you will need to take some time to get all the information you will need to request it.
+<br>
+<br>
+Resources: [GEAR](https://ea.gsa.gov/#!/itstandards)
+
+### Gather
+You will need several pieces of information when completing a software request. Things like purpose, cost, and download link among others. Follow [this checklist](https://docs.google.com/a/gsa.gov/spreadsheets/d/1zIyZGdmEuWUeuko7yEaupETz2YGJQpOumHwiUntul64/edit?usp=sharing){:target="_blank"} to be sure you've gathered all the info you will need.
+<br>
+<br>
+Resources: Software product checklist - ([Google Sheet](https://docs.google.com/a/gsa.gov/spreadsheets/d/1zIyZGdmEuWUeuko7yEaupETz2YGJQpOumHwiUntul64/edit?usp=sharing), [.pdf]({{ site.baseurl }}/assets/downloads/software-request-checklist.pdf))
+
+### Request
+Once you have gathered the info you will need on the software, it's time to put in your request. A quick way to do this is to select the Submitting a Service Catalog, [Request for Software](https://gsa.service-now.com/GSA_Self-Service/software.do) option in ServiceNow.
+<br>
+<br>
+Resources: [ServiceNow](https://gsa.service-now.com/GSA_Self-Service/software.do)
+
+#### Non-Standard Software Requests
+If the software you are requesting is not listed as Approved on the IT Standards List in GEAR, it will need to go through the non-standard request approval process. This involves 2 additional reviews where the software is evaluated for Security and Accessibility requirements. After these reviews are complete, the Technology Standards Committee (TSC) will provide their recommendation as to whether or not the software should be added to the IT Standards list. Finally, the Chief Technology Officer (CTO) will approve or reject the request. If approved, the software will be added to the IT Standards List in Gear. This process can take quite a bit longer than requesting software that is already approved in GEAR. Expect 4-6 weeks for software to be completely evaluated.
+
+#### Pilot Software Requests
+If the feasibility or applicability of the software is not known or not yet proven, you may request to evaluate the software on a trial basis. This type of request is know as a Pilot. Pilot software must never be on a production network or contain sensitive federal data and the cost for a Pilot may not exceed $25k.
```

Comment:
Suggestions:

* Say something like "the advantage is, the review process is simpler/faster"
* Make these criteria a list
* Make this section [plainer-language](https://content-guide.18f.gov/plain-language/)
    * For example, "If the feasibility or applicability of the software is not known" could be changed to "If you want to try a piece of software but aren't sure if you want it to go through a full review yet, ..."